doc/source/installation.rst

```diff
@@ -14,12 +14,13 @@ Installation
 1. Install http://github.com/tendrl/bridge_common
 2. Install http://github.com/tendrl/gluster_bridge
 
-    At the command line::
+3. At the command line::
 
+    $ yum install python-devel gcc # Install dependencies
```

Comment:
I dont think we should have distro specific details in install files.

Either make install files for centos, fedora, ubuntu, or add all those details here